From patchwork Fri Oct 25 01:54:36 2019
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Stephen Rothwell <sfr@canb.auug.org.au>
X-Patchwork-Id: 11211197
Return-Path: <SRS0=InyR=YS=vger.kernel.org=kvm-owner@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 2E07E14ED
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri, 25 Oct 2019 01:55:18 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [209.132.180.67])
	by mail.kernel.org (Postfix) with ESMTP id 0C71F206DD
	for <patchwork-kvm@patchwork.kernel.org>;
 Fri, 25 Oct 2019 01:55:18 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (2048-bit key) header.d=canb.auug.org.au header.i=@canb.auug.org.au
 header.b="rHSq/xld"
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S2389552AbfJYBy7 (ORCPT
        <rfc822;patchwork-kvm@patchwork.kernel.org>);
        Thu, 24 Oct 2019 21:54:59 -0400
Received: from ozlabs.org ([203.11.71.1]:39295 "EHLO ozlabs.org"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S2389488AbfJYBy4 (ORCPT <rfc822;kvm@vger.kernel.org>);
        Thu, 24 Oct 2019 21:54:56 -0400
Received: from authenticated.ozlabs.org (localhost [127.0.0.1])
        (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
         key-exchange ECDHE (P-256) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
        (No client certificate requested)
        by mail.ozlabs.org (Postfix) with ESMTPSA id 46znHK2bcBz9sP3;
        Fri, 25 Oct 2019 12:54:53 +1100 (AEDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=canb.auug.org.au;
        s=201702; t=1571968493;
        bh=7C+Sf4WcmsYMuwrmETgFQmmDGqQheurLBFa+StuqxPg=;
        h=Date:From:To:Cc:Subject:From;
        b=rHSq/xldZ2Ia1Ech2xZeEfzwY2F8dj0M6TCPtPrHcuUHpRyPvxiqpGn0e/pS/wG34
         IFKmqbfIXfio3vs/ry0EZ0NK/gUkZRs+FX2HDTXbn3oy4S1IJnby/MMo75lGAz9pUQ
         dC3pZu+po47WyqPVvb+9GeF45EwpehhDBnIu5I9r0v8iMB+Jy8gtmK14tyhzU3sZ3Q
         0yx40B71AAoHweOlQSGk4wTAflEyqVdxV4H19AJixQ5YSDMyUeOLs8BJcZ0FxcCgtH
         jn/z+qKh+tCSjxgK3D9Zb7LfoqzC5PG0zaGBbBAlZFf8HeFddFeZXZaxse1/WHCgRR
         NmgXBAMvezemw==
Date: Fri, 25 Oct 2019 12:54:36 +1100
From: Stephen Rothwell <sfr@canb.auug.org.au>
To: Paul Mackerras <paulus@ozlabs.org>,
        Christoffer Dall <cdall@cs.columbia.edu>,
        Marc Zyngier <marc.zyngier@arm.com>
Cc: Linux Next Mailing List <linux-next@vger.kernel.org>,
 Linux Kernel Mailing List <linux-kernel@vger.kernel.org>,
 Fabiano Rosas <farosas@linux.ibm.com>, Paolo Bonzini <pbonzini@redhat.com>,
 Radim =?utf-8?b?S3LEjW3DocWZ?= <rkrcmar@redhat.com>,
 KVM <kvm@vger.kernel.org>
Subject: linux-next: manual merge of the kvm-ppc tree with the kvm-arm tree
Message-ID: <20191025125436.6c85dbfe@canb.auug.org.au>
MIME-Version: 1.0
Sender: kvm-owner@vger.kernel.org
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Hi all,

Today's linux-next merge of the kvm-ppc tree got a conflict in:

  include/uapi/linux/kvm.h

between commits:

  c726200dd106 ("KVM: arm/arm64: Allow reporting non-ISV data aborts to userspace")
  da345174ceca ("KVM: arm/arm64: Allow user injection of external data aborts")

from the kvm-arm tree and commit:

  1a9167a214f5 ("KVM: PPC: Report single stepping capability")

from the kvm-ppc tree.

I fixed it up (see below) and can carry the fix as necessary. This
is now fixed as far as linux-next is concerned, but any non trivial
conflicts should be mentioned to your upstream maintainer when your tree
is submitted for merging.  You may also want to consider cooperating
with the maintainer of the conflicting tree to minimise any particularly
complex conflicts.

diff --cc include/uapi/linux/kvm.h
index fb47c0fad6a6,ce8cfcc51aec..000000000000
--- a/include/uapi/linux/kvm.h
